modules/swagger-codegen/src/main/resources/go-server/main.mustache

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	sw "./{{apiPath}}"
```

Comment:
all the generated code (except main.go) will be in that folder, so this is better than fully qualified path, (I expect people copy the generated code to their own project)